### ISSUE TITLE
[Tarea 1] Validez Mazo Kane.txt

### ISSUE THREAD

**Issue by tomasalvarez13** [IIC2113-2022-2/Syllabus#5]
Hola! Una pregunta, en el  mazo Kane.txt hay 6 cartas Arm Bar, y esta carta no es del tipo SetUp. El mazo esta malo o hay alguna condición que este pasando por alto?

**Comment by RodrigoToroIcarte** [comment]
Sí. Ese mazo estaba malo. Lo arreglé hace un par de días. Te recomiendo bajar la última versión del zip que hay en canvas. Ya vamos en el cartas_v7.zip :P. Ahí el mazo de KANE debería estar bien.
